TmdbEasy.Tests/V3TmdbEasyClientTests.cs

```diff
@@ -23,63 +22,5 @@ public void NullOptions_Throws_ArgumentNullException()
 
             Assert.Throws<ArgumentNullException>(() => new TmdbEasyClientv3(_jsonDeserializer, options));
         }
-
-        [Test]
-        public void OptionsWithSsl_Off_SetsCorrectVersionOfUrl_NoSsl()
-        {
-            var options = new TmdbEasyOptions()
-            {
```

Comment:
To test the correct SSL option, i would suggest that the `Request.GetUri()` method should return the absolute URL instead of the relative one (I didn't changed that because we didn't discussed it).

Reply:
It does indeed return the absolute path, unless I have misunderstood what you mean by this.

Reply:
Right now the `GetUri()˙ returns relative path. If i edit the code to return absolute path:
``` diff
public Request(TmdbEasyOptions options)
{
-    _requestBuilder = new StringBuilder();
+    _requestBuilder = new StringBuilder(options.BaseUri);
     _options = options;
}
```
a lot of Request test case will fail for some reason. *I will make a quick debugging to investigate this.*

Reply:
That change is what was originally on my branch. That should work. Its likely some tests need updating. Ill give it a quick look over

Reply:
I checked out your latest commit and ran tests, all passed. What failures are you seeing?

Reply:
They passed because i try to never commit with failing tests. If You initialize the StringBuilder with the `options.BaseUri` value they will fail.

Reply:
Gotcha, resolving this as its no longer a requirement [see review comment below](https://github.com/tonykaralis/TMdbEasy/pull/20#discussion_r462570683)